specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,2962 @@
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The App Component is deleted."
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete an app component": {
+            "$ref": "./examples/DeleteAppComponent.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "AppComponent"
+        ],
+        "summary": "Get app Component details by unique name.",
+        "operationId": "AppComponent_GetByTestOrTestRun",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "name",
+            "description": "Unique name of the app component, must be a valid URL character ^[a-z0-9_-]*$.",
+            "required": true,
+            "type": "string",
+            "maxLength": 50,
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The requested app components",
+            "schema": {
+              "$ref": "#/definitions/AppComponentsMap"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get an app component by a test or test run": {
+            "$ref": "./examples/GetAppComponentByTestOrTestRun.json"
+          }
+        }
+      }
+    },
+    "/appcomponents": {
+      "get": {
+        "tags": [
+          "AppComponent"
+        ],
+        "summary": "Get app component for a test or a test run by unique its name.",
+        "operationId": "AppComponent_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/TestIdQueryParameter"
+          },
+          {
+            "in": "query",
+            "name": "testRunId",
+            "description": "Required testRunId, if testId field is not provided",
```

Comment:
Is there no way to simply get a list of appComponents? If not, why not?

Reply:
added more context on above comment